.eslintrc.json

```diff
@@ -1,4 +1,17 @@
 {
 	"root": true,
-	"extends": "./node_modules/eslint-config-mdcs/index.js"
+	"extends": "./node_modules/eslint-config-mdcs/index.js",
+  "overrides": [
+		{
+			"files": ["*.ts"],
+			"parser": "@typescript-eslint/parser",
+			"plugins": [
+				"@typescript-eslint"
+			],
+			"rules": {
+				"no-unused-vars": ["error", { "args": "none" }],
+				"indent": [2, 2]
+			}
+		}
+	]
```

Comment:
More indentation fixes to make. I use `editorconfig` with vs code to automatically configure indentation.

Reply:
Thanks, I'll check